### DISCUSSION TITLE
Backup of Data

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#425] [Q&A] [answered]
Is there a possibilty to make a Export/Backup of whole Data and Settings of Event Made Easy?
So if i have to set up a Page new i can import all result back to a new Website?

**Comment by liedekef** [comment, marked as answer]
Most EME data are in seperate database tables (starting with "eme_") that you can export/import using your favourite db tool.
The generic EME settings are stored in the regular WP options table (name also starts with "eme_").
